services/src/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java

```diff
@@ -126,10 +130,12 @@ public void shutdown() {
     public CompletableFuture<List<CliReply>> setLogicalPort(LogicalPortDto port) {
         Objects.requireNonNull(port.getLogicalPortNumber(), "Logical port number must not be null");
```

Comment:
what if logical port itself is null

Reply:
This case will be handled, and return a correct error